src/avram/tasks/db/drop.cr

```diff
@@ -3,8 +3,11 @@ require "colorize"
 class Db::Drop < BaseTask
   summary "Drop the database"
 
+  def initialize(@quiet : Bool = false)
+  end
+
   def run_task
-    Avram::Migrator::Runner.drop_db
+    Avram::Migrator::Runner.drop_db(@quiet)
```

Comment:
I don't think this will work just yet. That method currently doesn't take any arguments. https://github.com/luckyframework/avram/blob/master/src/avram/migrator/runner.cr#L57 You'll have to update the method just like the create_db https://github.com/luckyframework/avram/blob/master/src/avram/migrator/runner.cr#L67 and then it has to actually be implemented.

Reply:
aaah yes, that makes a lot of sense 😁 Thank you for the tip!